venues/MIDL.io/2019/Conference/Full/process/submissionProcess.js

```diff
@@ -28,6 +28,38 @@ function () {
             readers: invitation.reply.readers
         }
    };
+   var commentInvitation = {
+        id: CONFERENCE_ID + '/-/Paper'+note.number+'/Pams_Comment',
```

Comment:
Pams_Comment?

Reply:
Oops!  I'm still working on the comments as I'm not clear on who should be able to see them.  I'll get that out of there.